File: ws/salazar/u2/ws32PQUIZ/Coop.py
import Chicken
class Coop:
    def __init__(self,id,chickens):
        self.id = int(id)
        self.chickens = Chicken(chickens)
        
    def get_id(self):
        return self.id
        
    def set_id(self, id):
        self.id = id
        
    def add(self,chicken):
        self.chickens.append(chicken)
        
    def remove(self,chickenId):
        for chicken in self.chickens:
            if(chicken.id==chickenId):
                self.chickens.pop(chicken)
                
    def resetIteration(self):
        self.chickens.clear()
                
    def next():
        return Chicken

File: ws/suquillo/u2/ws32Pquiz/ChickenFarmSimulator/ChickenFarmSimulator/ChickenFarmModel/Coop.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography.X509Certificates;
using System.Text;
using System.Threading.Tasks;

namespace ChickenFarmSimulator.ChickenFarmModel
{
    internal class Coop
    {
        public int id { get; set; }
        public Chicken[] chikens { get; set; }
        public void add(Chicken chicken) { }
        public void remove(int chickenId) { }
        public void resetIteration() { }
        public Chicken next()
        {
            return new Chicken();
        }
    }
}
